jerry-core/include/jerryscript-core.h

```diff
@@ -349,6 +349,11 @@ jerry_value_t jerry_acquire_value (jerry_value_t value);
 void jerry_release_value (jerry_value_t value);
 
 /**
+ * Get the 'real' value of a jerry_value
+ */
+jerry_value_t jerry_get_value (jerry_value_t value);
```

Comment:
Should be declared with the other error-related functions (around lines 295-297)?